[2024-05-17T20:51:02.636+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-17T20:51:02.685+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_pipelines_2.read_data manual__2024-05-17T12:31:17.083416+00:00 [queued]>
[2024-05-17T20:51:02.698+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_pipelines_2.read_data manual__2024-05-17T12:31:17.083416+00:00 [queued]>
[2024-05-17T20:51:02.701+0000] {taskinstance.py:2306} INFO - Starting attempt 25 of 25
[2024-05-17T20:51:02.722+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): read_data> on 2024-05-17 12:31:17.083416+00:00
[2024-05-17T20:51:02.734+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=11088) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-17T20:51:02.737+0000] {standard_task_runner.py:63} INFO - Started process 11098 to run task
[2024-05-17T20:51:02.736+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_pipelines_2', 'read_data', 'manual__2024-05-17T12:31:17.083416+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/data-ingestion-dag.py', '--cfg-path', '/tmp/tmpqak8qlqp']
[2024-05-17T20:51:02.738+0000] {standard_task_runner.py:91} INFO - Job 91: Subtask read_data
[2024-05-17T20:51:02.805+0000] {task_command.py:426} INFO - Running <TaskInstance: data_ingestion_pipelines_2.read_data manual__2024-05-17T12:31:17.083416+00:00 [running]> on host e512d0910ba5
[2024-05-17T20:51:02.935+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_pipelines_2' AIRFLOW_CTX_TASK_ID='read_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-17T12:31:17.083416+00:00' AIRFLOW_CTX_TRY_NUMBER='25' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-17T12:31:17.083416+00:00'
[2024-05-17T20:51:02.937+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-17T20:51:02.961+0000] {logging_mixin.py:188} INFO - file_path in read_data is  /opt/***/dags/raw-data/data_part_1.csv
[2024-05-17T20:51:02.978+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-17T20:51:02.979+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-17T20:51:02.992+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_ingestion_pipelines_2, task_id=read_data, run_id=manual__2024-05-17T12:31:17.083416+00:00, execution_date=20240517T123117, start_date=20240517T205102, end_date=20240517T205102
[2024-05-17T20:51:03.033+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-17T20:51:03.046+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
